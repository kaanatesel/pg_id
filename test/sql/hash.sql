SELECT mgid_hash('507f1f77bcf86cd799439011');
SELECT mgid_hash('507f191e810c19729de860ea');
SELECT mgid_hash('507f191e810c19729');
SELECT mgid_hash_extended('507f1f77bcf86cd799439011');
SELECT mgid_hash_extended('507f191e810c19729de860ea');
SELECT mgid_hash_extended('507f191e810c19729');