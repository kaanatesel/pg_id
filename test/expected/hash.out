SELECT mgid_hash('507f1f77bcf86cd799439011');
  mgid_hash  
-------------
 -1713493780
(1 row)

SELECT mgid_hash('507f191e810c19729de860ea');
  mgid_hash  
-------------
 -1348366109
(1 row)

SELECT mgid_hash('507f191e810c19729');
ERROR:  invalid input syntax for type mgid: "507f191e810c19729"
LINE 1: SELECT mgid_hash('507f191e810c19729');
                         ^
SELECT mgid_hash_extended('507f1f77bcf86cd799439011');
 mgid_hash_extended 
--------------------
        -1713493780
(1 row)

SELECT mgid_hash_extended('507f191e810c19729de860ea');
 mgid_hash_extended 
--------------------
        -1348366109
(1 row)

SELECT mgid_hash_extended('507f191e810c19729');
ERROR:  invalid input syntax for type mgid: "507f191e810c19729"
LINE 1: SELECT mgid_hash_extended('507f191e810c19729');
                                  ^
